layouts/partials/meta.html

```diff
@@ -1,6 +1,34 @@
+{{ $url     := .Permalink }}
+{{ $title   := cond .IsHome site.Title .Title }}
+{{ $type    := cond .IsHome "website" "article" }}
+{{ $desc    := .Param "description" }}
+{{ $twitter := printf "@%s" site.Params.social.twitter_handle }}
+{{ $img     := "img/logos/grpc-icon-color.png" | absURL }}
+{{ $imgAlt  := printf "%s color logo" site.Title }}
```

Comment:
Why the "color logo" text? Shouldn't the title only be better, or possibly even the description?

Reply:
The image `alt` tag is meant to provide a description of the image itself, not of the document or site.

Reply:
Right (though the image is meant to represent the content). Ok well, maybe drop "color". (Yes, this is just a nit).